services/QuillLMS/Gemfile.lock

```diff
@@ -1,3 +1,10 @@
+GIT
+  remote: https://github.com/cm-studios/omniauth-clever
+  revision: 9b32b15a3daca46aee1011e18e578fd61ec9c23e
+  specs:
+    omniauth-clever (1.1.1)
```

Comment:
So this fork is 8 commits ahead and 14 commits behind the gem we were on (this would be a change to a fork and revision to from `1.2.2` -> `1.1.1`). 

There seems to be some notable changes in urls and auth methods:
https://github.com/cm-studios/omniauth-clever/compare/master...Clever:omniauth-clever:master#diff-cd2bee9e4d8f887976e3c4942853cfbe6de360f219dcd7f8c53c1f0b3f835228

This might be worth looking at. We mind need our own fork where we incorporate this forks changes and update to `1.2.2` (there may also be a fork that already did that)

Reply:
There is https://github.com/Clever/omniauth-clever/compare/master...Pioneer-Valley-Books:omniauth-clever:master